front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/utils.ts

```diff
@@ -1,21 +1,65 @@
 import type { PathLevel, HoveredItem } from '@osrd-project/ui-charts';
 
 import { PATH_COLORS } from 'modules/simulationResult/consts';
-import type { TrainId } from 'reducers/osrdconf/types';
-import { formatEditoastTrainIdToTrainScheduleId } from 'utils/trainId';
+import type { OccurrenceId, TrainId } from 'reducers/osrdconf/types';
```

Comment:
seems to be not used.
```suggestion
import type { TrainId } from 'reducers/osrdconf/types';
```